suite-native/module-send/src/screens/SendAddressReviewScreen.tsx

```diff
@@ -0,0 +1,151 @@
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
UI nit:
<img src="https://github.com/user-attachments/assets/457a4b23-e1c1-4192-9601-318eef4bbcab" width=350 />

Reply:
fixed by removing redundat `position: 'absolute'` from the footer component (it is alreadt applied inside of the `Screen` definition. 8867fec3ab